main.nf

```diff
@@ -501,7 +490,6 @@ process bowtie2 {
 
     input:
     file reads from trimmed_reads_bowtie2
-    set val(name) from name_for_bowtie2
 
     output:
     file '*.bowtie2.bam' into bowtie2_bam
```

Comment:
Hi, I kept the long prefix cleanup script there because they are using files directly from Trim_galore, which will keep the original file name such as P1001_101.R1.fq.gz or something similar. I want to have the prefix be to simply the sample ID "P1001_101"